packages/mobile/package-lock.json

```diff
@@ -6300,7 +6300,7 @@
       "version": "file:../web",
       "requires": {
         "@audius/libs": "1.2.104",
-        "@audius/stems": "0.3.27",
```

Comment:
Double check the package-lock.json changes, seems surprising to have this much with just a single babel-plugin install?